packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -281,7 +281,17 @@ function VM-Install-Shortcut{
     )
     $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
     $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+    $extension = [System.IO.Path]::GetExtension($executablePath)
 
+    # If the shortcut is for a .py, we should set the shortcut icon to be Python (requires Python to be installed)
```

Comment:
does this fail if Python is not installed?

Reply:
Hmmm. The `$pythonPath = (Get-Command python).Source` would likely error out. 

I can add a better check here to mitigate this, like:
```
If (($extension -eq ".py") -and (Get-Command python)) {
    $iconLocation = (Get-Command python).Source
}
Else {
???
}
```
Not sure what we should do if it isn't installed...Also, would we ever reach this if it wasn't if we're controlling the calls to `VM-Install-Shortcut`?